src/main/java/com/ibm/watson/developer_cloud/util/CredentialUtils.java

```diff
@@ -32,6 +32,35 @@
  * The Class CredentialUtils.
  */
 public final class CredentialUtils {
+  /**
+   * A util class to easily store service credentials.
+   * 
+   */
+  public static class ServiceCredentials {
+    private String password;
+    private String username;
+
+    public ServiceCredentials(String username, String password) {
+      this.username = username;
+      this.password = password;
+    }
+
+    public String getPassword() {
+      return password;
+    }
+
+    public String getUsername() {
+      return username;
+    }
+
+    public void setPassword(String password) {
```

Comment:
Can you remove the `set` and make it immutable? I think the constructor is enough and you will use this class as a container mostly